### ISSUE TITLE
Description for schema fields

### ISSUE THREAD

**Issue by artreven** [plangrid/flask-rebar#128]
Hi there. Thanks for the great project!

I do not find a way to provide description for different fields in my request schema. Is it supported?

```
class GetRelationsSchema(RequestSchema):
    """
    Some request description
    """
    text = fields.String(
        required=True,
        description="some field description")
```

results in 
![swagger_field](https://user-images.githubusercontent.com/1529120/65025651-673d2580-d937-11e9-9ab2-0146a5e29e30.png)

I use `SwaggerV3Generator`.

**Comment by artreven** [comment]
I think I found the source of the problem.
The generated JSON description has this part:
```
"parameters": [
          {
            "in": "query",
            "name": "text",
            "required": true,
            "schema": {
              "description": "Seom field description",
              "type": "string"
            }
          }
        ],
```
however, OpenAPI3 spec expects the description to be 1 level higher, see https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#fixed-fields-6

**Comment by airstandley** [comment]
Technically SwaggerV3Generator follows 3.0.2; not that it matters in this case.

I think this ended up this way because the generated JSON description is not 'necessarily' wrong.

"SchemaObject" can have a description https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#schema-object

It's not entirely clear to me whether the intention of passing `description` metadata to Marshmallow.field.Field was to describe the Schema the field represented or to describe the property/parameter.

**Comment by airstandley** [comment]
Based on @barakalon 's endorsement of the ticket I'm going to assume it was intended to be the parameter description 😄

**Comment by RookieRick** [comment]
Fixed by #129
